### PR TITLE
ticket #2. Examples now work.

### PR DESCRIPTION
**Pull request by rupertford**
The examples, other than PSyclone (which is under development) now work. To test, set you Python path to the src directory, then run one of the examples e.g. cd examples/longbow_gromacs; python ./gromacs.py. You should see a list of outputs representing all options specified in the Python input file.

### CODE REVIEW

**Comment by arporter** [comment]
Not sure exactly what you'd like me to review :-)
Just the commits under this issue presumably?

File: src/melody/inputs.py [hunk 1]
```diff
@@ -1,93 +1,74 @@
 ''' '''
 
-class Fixed(object):
+class Base(object):
```
Comment (arporter) [inline src/melody/inputs.py]: I don't think 'Base' is a good name for a class :-) How about InputBase or just Input?

**Review by arporter — CHANGES_REQUESTED** [review]
I can run the gromacs example without problems.

Please add doc-strings and comments to the code touched in this issue!
Otherwise, I think it looks fine (I'm not going to try and understand everything here).

**Comment by rupertford** [comment]
I've changed the class name as (correctly) suggested. I've made a separate ticket to make the code fully pep8 and (reasonably) pylint compliant which will include adding doc strings throughout the source. So, can you skip that part of the review for the timebeing - pretty please.

**Review by arporter — APPROVED** [review]
Base class name has been changed to Input - good.
All OK.

**Comment by rupertford** [comment]
Merging to Master

**Pull request merged**